android/specs/local_test.js

```diff
@@ -0,0 +1,44 @@
+const { Builder, By, until } = require("selenium-webdriver");
+var assert = require('assert');
+
+var buildDriver = function() {
+  return new Builder()
+    .usingServer('http://127.0.0.1:4723/wd/hub')
+    .build();
+};
+
+describe('BrowserStack Local Testing', function() {
+  this.timeout(0);
```

Comment:
This means test will be marked as failed always right? Because timeout is called before test is completed?

Reply:
this is to disable timeouts.